util/dvsim/Deploy.py

```diff
@@ -39,6 +39,7 @@ class Deploy():
     print_interval = 5
     max_parallel = 16
     max_odirs = 5
+    max_new_jobs = 20
```

Comment:
nit: rename to `slot_limit` and add comment indicating that this is the max jobs dispatched in one go.

Reply:
> Wouldn't it be easier to set up a job array for this? Then you can submit 1000 jobs at once or whatever and you won't have a whole load of `bsub` processes sitting around. At least, I think that's how you're supposed to do it with LSF.

Agree - setting up dvsim to create job arrays is the right way to go. This is a quick workaround until we get that set up.